README.md

```diff
@@ -77,6 +77,7 @@ fastify.register(require('fastify-rate-limit'), {
 - `whitelist`: array of string of ips to exclude from rate limiting. It can be a sync function with the signature `(req, key) => {}` where `req` is the Fastify request object and `key` is the value generated by the `keyGenerator`. If the function return a truthy value, the request will be excluded from the rate limit.
 - `redis`: by default this plugins uses an in-memory store, which is fast but if you application works on more than one server it is useless, since the data is store locally.<br>
 You can pass a Redis client here and magically the issue is solved. To achieve the maximum speed, this plugins requires the use of [`ioredis`](https://github.com/luin/ioredis)
+- `store`: a custom store to track requests and rates, for an example custom store please see the [`example/`](https://github.com/fastify/fastify-rate-limit/tree/master/example) folder
```

Comment:
Could we have a minimal store-example here?
The `CustomStore` in test is perfect

Reply:
Added in the README an example as well as pointing them to the examples folder.